### ISSUE TITLE
Named mobs stuck in chairs

### ISSUE THREAD

**Issue by Echoweaver** [Nuchaz/BiblioCraft#564]
A cow I had named got loose and wandered into my house and got stuck on my chair. I removed the chair and the table, still stuck. Couldn't even kill her. Someone tipped me off that I could get the cow to move using a lead. And that's fine, but I now can't use any chairs because my tamed wolf gets stuck in one every time I enter the house. 

I read up on the Bibliocraft webpage and was shocked to find that the named mobs sitting in chairs was part of the design, but it can't be working as intended.

ETA: I noticed the Doggy Talents issue mentioning shift-click to get mobs out of chairs. I tried and shift-click didn't seem to do anything (Minecraft 1.12.2)

**Comment by Nuchaz** [comment]
You have to shift click with an empty hand on the seat to dismount a mob. I just tested in 1.12.2 and it seems to be working fine. You get about 4 seconds before the seat will try to make the mob sit again. Perhaps I'll consider adding a config option to disable that feature for folks who find it undesirable. Or perhaps I should just blacklist wolves. Even better would be a GUI for the seat that let the user enable / disable mob sitting. That might be a better option, then you can set it on a per seat basis. I'll put some thought into this idea for a future release.

**Comment by acook** [comment]
I'd request a config option for excluding mobs based on their entity identifier. So excluding all dogs would be one that might be a good default but some people might want to add say cows as well but still want villagers. 

A feature beyond that might be to let people use a lead to put entities into chairs to get around the exclude list, but I'd say that's lower priority imho.

**Comment by talayawhite** [comment]
I have this same problem, except it's a villager floating in the air now, and you can't use leashes on villagers. Help?

**Comment by SanjivanM** [comment]
Yeah, like @Nuchaz said, shift clicking on the seat itself (not the back, but in the hitbox of the seat) with an empty hand will dismount the mob.  Tried it with a sheep and an MCA villager.  I'd suggest adding a cooldown for the seat before it attempts to pull a mob in, because there's nothing better than your dog already waiting at the dinner table :P

**Comment by CovertJaguar** [comment]
I'll just add that the teleport command works to remove them from the seat as well.
